internal/host/transition_test.go

```diff
@@ -2757,41 +2757,59 @@ var _ = Describe("Refresh Host", func() {
 			models.HostStatusInstallingPendingUserAction,
 			models.HostStatusResettingPendingUserAction,
 		} {
-			srcState := srcState
```

Comment:
Added all stages to this test (nothing else changed) because it failed to detect this bug without them